ChecksFile.wl

```diff
@@ -61,61 +61,30 @@ On[Assert]
-  1.04928*10^-21}, {1.12028*10^-15, 1.05019*10^-21}, {1.23067*10^-15, 
-  1.05102*10^-21}, {1.35194*10^-15, 1.05178*10^-21}, {1.48516*10^-15, 
-  1.05247*10^-21}, {1.63151*10^-15, 1.05309*10^-21}, {1.79228*10^-15, 
-  1.05366*10^-21}, {1.9689*10^-15, 1.05418*10^-21}, {2.16292*10^-15, 
-  1.05465*10^-21}, {2.37605*10^-15, 1.05508*10^-21}, {2.61019*10^-15, 
-  1.05547*10^-21}, {-2.61019*10^-17, 1.44396*10^-21}, {-3.3261*10^-17,
-   1.36214*10^-21}, {-4.23837*10^-17, 
-  1.29781*10^-21}, {-5.40084*10^-17, 
-  1.24715*10^-21}, {-6.88216*10^-17, 
-  1.20722*10^-21}, {-8.76976*10^-17, 
-  1.17574*10^-21}, {-1.11751*10^-16, 
-  1.15093*10^-21}, {-1.42401*10^-16, 
-  1.13139*10^-21}, {-1.81459*10^-16, 1.116*10^-21}, {-2.31228*10^-16, 
-  1.10389*10^-21}, {-2.94648*10^-16, 
-  1.09436*10^-21}, {-3.75463*10^-16, 
-  1.08687*10^-21}, {-4.78443*10^-16, 
-  1.08098*10^-21}, {-6.09668*10^-16, 
-  1.07635*10^-21}, {-7.76884*10^-16, 
-  1.07272*10^-21}, {-9.89964*10^-16, 
-  1.06986*10^-21}, {-1.26149*10^-15, 
-  1.06762*10^-21}, {-1.60748*10^-15, 
-  1.06586*10^-21}, {-2.04837*10^-15, 
-  1.06448*10^-21}, {-2.61019*10^-15, 1.06339*10^-21}};
+OldData={{3.943351801881396`*^-20,0},{6.403122930332929`*^-20,0},{1.0397242072440586`*^-19,6.475185304707499`*^-23},{1.6882799828943322`*^-19,0},
+{2.7413897654617433`*^-19,1.2805250574405962`*^-19},{4.451404934206825`*^-19,2.149878747929582`*^-19},{7.228087788874928`*^-19,2.877126337699488`*^-19},
+{1.173680082038912`*^-18,3.418162146974343`*^-19},{1.9057944164639052`*^-18,3.5031848436345418`*^-19},{3.0945846431298252`*^-18,2.6089021800081186`*^-19},
+{5.02491456096483`*^-18,7.0760561598301275`*^-22},{8.159339380505439`*^-18,7.514263464205905`*^-22},{1.3248945493211215`*^-17,7.789552045160011`*^-22},
+{2.1513329510655564`*^-17,7.961350592910255`*^-22},{3.493284404190471`*^-17,8.068054415566711`*^-22},{5.672313958895247`*^-17,8.134119405806582`*^-22},
+{9.210571463829571`*^-17,8.174940729310282`*^-22},{1.4955911697601837`*^-16,8.200132465051396`*^-22},{2.4285061527926315`*^-16,8.215666277345043`*^-22},
+{3.9433518018813854`*^-16,8.225240355557669`*^-22},{-3.943351801881396`*^-20,1.2187126603736879`*^-22},{-1.0972610632535474`*^-19,3.174638959109932`*^-20},
+{-3.053194088231941`*^-19,5.832887809619426`*^-20},{-8.495693916972988`*^-19,1.851851392271892`*^-21},{-2.3639772986947037`*^-18,1.1000927210473065`*^-21},
+{-6.577907259087188`*^-18,9.189580059662592`*^-22},{-1.830341769061953`*^-17,8.575191336733617`*^-22},{-5.0930346987557535`*^-17,8.360035843249597`*^-22},
+{-1.4171671586789936`*^-16,8.283420407361107`*^-22},{-3.9433518018813854`*^-16,8.255976606410335`*^-22}};
 kdebug=Module[{data,ri=200,testE=20,hi=-1,k=0,kvar},
 file=inpath<>"1D_withV_withPairBrems_DO.h5";
 (*SCalcScale[ImportData[inpath<>file<>".h5"],ri,testE,hi,0][[3]]//MatrixForm*)
 (*buildkGrid[ImportData[inpath<>file<>".h5"],ri,testE,hi,40]*)
-kAdapt[file,ri,ri,testE,hi,10,"xflavor"-> False]
+kAdapt[file,ri,ri,testE,hi,20,"xflavor"-> False]

```

Comment:
Should this be testE?

Reply:
You mean the 10 to 20 change? That's the length of the k grid

Reply:
Oh, I see. Then is there a way to automate that so it doesn't have to be entered by hand?

Reply:
It can be added to the module definitions in the same way ri, testE and hi are.